### PR TITLE
Web Share: Manual tests for file sharing

### PR DESCRIPTION
**Pull request by ewilligers**
Check that an image file and a collection of text files can be shared.

### CODE REVIEW

**Comment by ewilligers** [comment]
@raymeskhoury This complements the automated tests.

**Comment by ewilligers** [comment]
@marcoscaceres  please review

**Review by raymeskhoury — APPROVED** [review]

File: web-share/resources/manual-helper.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ function setupManualShareTest(expected_share_data) {
   item = document.createElement('li');
   list.appendChild(item);
   item.innerText = `url = "${url}"`;
+  if (!!files) {
```
Comment (marcoscaceres) [inline web-share/resources/manual-helper.js]: Why not just? 
```suggestion
  if (files) {
```

File: web-share/resources/manual-helper.js [hunk 2]
```diff
@@ -50,6 +50,11 @@ function setupManualShareTest(expected_share_data) {
...
+    item = document.createElement('li');
+    list.appendChild(item);
+    item.innerText = `files = ${files.length} file(s)`;
```
Comment (marcoscaceres) [inline web-share/resources/manual-helper.js]: Worth showing the content of each? I guess just knowing you got them might be enough.

**Review by marcoscaceres — APPROVED** [review]
This looks fine to me...  little question/nit.

File: web-share/resources/manual-helper.js (hunk 1 above)
Comment (ewilligers) [reply web-share/resources/manual-helper.js]: Changed.

File: web-share/resources/manual-helper.js (hunk 2 above)
Comment (ewilligers) [reply web-share/resources/manual-helper.js]: Now showing content.

**Review by marcoscaceres — APPROVED** [review]
Thanks for making the additional changes. Looks great.

**Pull request merged**